### ISSUE TITLE
How can I pass the tokens in the modal function lookup?

### ISSUE THREAD

**Issue by KMC07** [modal-labs/modal-examples#364]
If I am calling a modal function from a Django backend, is there any I can set the API tokens ID and Secret in the call of the function lookup? Because I would be using digital ocean apps for the Djano backend, which might not give me access to the console letting me set up the tokens.

Or do you have to set up the tokens beforehand instead of calling them programmatically?

**Comment by ekzhang** [comment]
Hi! You can set the auth tokens as environment variables: https://modal.com/docs/reference/modal.config

**Comment by KMC07** [comment]
Would I just set it in my django .env file? Because I am not sure if I can access the home directory since I would be using digital oceans apps

**Comment by ekzhang** [comment]
You don't need to modify the `.modal.toml` file if you set the environment variables.

**Comment by ekzhang** [comment]
> Would it work even if I set it in my Django `.env` file?

Yes, if. you're loading a `.env` file then that would set environment variables. (But also, it's not good software engineering practice to check authentication secrets into source control.)

**Comment by KMC07** [comment]
Thank you for the information. I appreciate the quick response. I would load the `.env`, but don't worry the `.env` is not included in the source control.
